camlidl.opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "camlidl"
+version: "1.11"
+synopsis: "Stub code generator for OCaml"
+description: """\
+CamlIDL is a stub code generator for OCaml. It automates the
+generation of C stub code required for the Caml/C interface, based on
+an MIDL specification. Also provides some support for Microsoft's COM
+software components."""
+maintainer: "Nicolas Berthier <m@nberth.space>"
+authors: "Xavier Leroy"
+license: [
+  "QPL-1.0 WITH OCaml-LGPL-linking-exception"
+  "LGPL-2.0-or-later WITH OCaml-LGPL-linking-exception"
+]
```

Comment:
For the record: the QPL is no longer used, CamlIDL as a whole is under the LGPL-2-0 with OCaml exception.